code/game/objects/effects/decals/Cleanable/misc_cleanables.dm

```diff
@@ -277,6 +277,36 @@
 	icon_state = "flour"
 	color = "#D5820B"
 	scoop_reagents = list("fungus" = 10)
+	no_clear = TRUE
+	var/timer_id
+
+/obj/effect/decal/cleanable/fungus/examine(mob/user)
+	. = ..()
+	if(no_scoop)
+		. += "<span class='notice'>There's really not a lot here, you probably couldn't harvest anything if you tried.</span>"
```

Comment:
```suggestion
		. += "<span class='notice'>There's not a lot here, you probably wouldn't be able to harvest anything useful.</span>"
```
Examine text should be concise IMO but this is pretty nitpicky

Reply:
I was having trouble wording that, appreciate it